book/sections/ogc_existingBuildingBlocks.md

```diff
@@ -1,3 +1,64 @@
-# Existing OGC building blocks for climate service
+# OGC Building Blocks for Climate Services
 
 Overview of birdhouse components
+
+```{figure} /media/Birdhouse_Schema.png
+:scale: 50%
+```
+
+<table class="c28">
```

Comment:
probably you wanted to copy the text ... but not the html ...

Reply:
OK. I ll do, when I find a quiet minute ;-)